src/_pytest/hookspec.py

```diff
@@ -185,15 +198,15 @@ def pytest_collection_modifyitems(session, config, items):
     """
 
 
-def pytest_collection_finish(session):
+def pytest_collection_finish(session: "Session"):
     """ called after collection has been performed and modified.
 
     :param _pytest.main.Session session: the pytest session object
     """
 
 
 @hookspec(firstresult=True)
-def pytest_ignore_collect(path, config):
+def pytest_ignore_collect(path, config: "Config"):
```

Comment:
`path: py.path.local`

Reply:
I prefer leaving annotating `path` out of this particular PR.